source/deployment-considerations/colocation.txt

```diff
@@ -0,0 +1,105 @@
+run Relational Migrator. Also note that the 
+:ref:`Kafka deployment model <kafka-intro>` provides greater 
+resilience and scalability and can be used with any of the 
+deployment options.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use case 
+     - Recommended environment
+
+   * - Smaller scale or debugging and testing jobs that run for 
+       less than four hours. 
+     - Laptop / Desktop / Docker
+
+   * - Larger scale production jobs that run for more than four 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to Run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If you're using Relational Migrator for an on-premises migration, 
+  run Relational Migrator in the same data center as the source 
+  database.
+
+- If you're using a cloud hosted database, run Relational 
+  Migrator on an EC2 instance or VM in the same 
+  :abbr:`VPC (Virtual Private Cloud)` as the source database.
+
+.. tip::
+
+   Check the sleep timeout setting for your operating system. If your 
+   machine goes to sleep during a migration, the sync job fails. 
+
+Network Considerations
+----------------------
+
+When using Relational Migrator, the system firewall on the machine or 
+server must allow inbound and outbound TCP traffic to the source
+database. The source database connection must also allow TCP traffic 
+from the IPV4 address of the machine or server running Relational
+Migrator.
+
+VPC and Cloud Networking
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Relational Migrator on an AWS EC2 instance, check the 
+security group and firewall configurations of the EC2 instance or VM. 
+Also check that the router table allows traffic to the database. If your 
+database is hosted on a public IP address, the EC2 
+instance needs a router table entry to an internet gateway within the 
+:abbr:`VPC (Virtual Private Cloud)`. For details on specific cloud 
+provider network configurations, see these pages:
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
+~~~~~~~~~~~~~~~~
+
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__ 
+is a solution that lets you establish private connections between 
+your VPC and various services. AWS private link makes network traffic 
+behave as though the connected systems are located within your own VPC.
+
+You can configure a `AWS private link endpoint <https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html>`__ 
+for usage with Relational Migrator. This setup 
+eliminates the need for an internet gateway or public IP addresses.
+
+System Hardware
```

Comment:
Somewhere (not necessarily this section), can we call out that migration is I/O intensive on the destination MongoDB cluster, and that scaling up the cluster can significantly reduce migration time?